labgrid/resource/udev.py

```diff
@@ -301,7 +301,12 @@ class RKUSBLoader(USBResource):
     def filter_match(self, device):
         match = (device.properties.get('ID_VENDOR_ID'), device.properties.get('ID_MODEL_ID'))
 
-        if match not in [("2207", "110a")]:
+        if match not in [("2207", "110a"), ("2207", "301a"),
+                         ("2207", "310c"), ("2207", "320b"),
+                         ("2207", "320a"), ("2207", "320c"),
+                         ("2207", "330a"), ("2207", "330c"),
+                         ("2207", "350a"), ("2207", "350b")
+                         ]:
```

Comment:
suggestion (non-blocking): please comment on the SoC model those represents

This would make it easier to spot what's supported and what's not.

330c is RK3399
350b is RK3588

I also have:

330d for PX30 (but I can add myself, not an issue)

c.f. https://github.com/rockchip-linux/rkdeveloptool/blob/master/99-rk-rockusb.rules for some model ID

https://elixir.bootlin.com/u-boot/v2024.10/source/drivers/usb/gadget/Kconfig#L69 probably could help too :)

Reply:
I agree with this.

Reply:
I didn't do that before, because I didn't know what the original VID/PID was for, but I can annotate them now. Thanks!